db/migrate/20210625083734_add_admit_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddAdmitToUser < ActiveRecord::Migration[6.1]
```

Comment:
Read about frozen_string_literal comment and what it does
https://stackoverflow.com/questions/37799296/ruby-what-does-the-comment-frozen-string-literal-true-do